dotcom-rendering/scripts/webpack/gu-stats-report-plugin.js

```diff
@@ -0,0 +1,144 @@
+const fetch = require('node-fetch');
+const os = require('os');
+const { exec } = require('child_process');
+
+class GuStatsReportPlugin {
+	constructor(config) {
+		this.buildName = config?.buildName;
+		this.project = config?.project;
+		this.team = config?.team;
+		this.sessionId = config?.sessionId;
+		this.buildCount = 0;
+		this.gitBranch = undefined;
+		this.gitHash = undefined;
+
+		this.fetchGitBranch();
```

Comment:
Could be worth leaving a note similar to [this](https://github.com/guardian/dotcom-rendering/pull/3848#discussion_r790571082) - although maybe it's obvious enough not to.